### ISSUE TITLE
Polygon (replace bounding boxes)

### ISSUE THREAD

**Issue by saanasum** [sentinel-hub/sentinelhub-py#37]
Is there a parameter for generating areas of interest as polygons? 

Like described for the http-request: https://www.sentinel-hub.com/faq/how-can-i-clip-image-specific-polygon

`from sentinelhub import Polygon` --> does not work, which indicates that it will not be as easy as with BBoxes

**Comment by AleksMat** [comment]
Hi,

at the moment `sentinelhub` Python package does not yet support usage of geometry parameter of Sentinel Hub services. We might add support for polygons some time in the future (when time permits).

Since the package is open-source we are also happy to accept any merge requests and code contributions by anyone.

**Comment by saanasum** [comment]
Hi,

many thanks for your quick reply. Due to being a beginner in python it might take a while until I will find a solution. However, as soon as I got it working, I will post it here for further discussion. 

Best
s.

**Comment by AleksMat** [comment]
Update about geometry and polygons: 

In newly released `sentinelhub` version `2.1.0` we added `CustomUrlParam.GEOMETRY` which can be used to specify geometry. E.g.:

```Python
wms_request = WmsRequest( ... , 
    custom_url_params={
        CustomUrlParam.GEOMETRY: "POLYGON((1292344.0 5205055.5, 1301479.5 5195920.0, 1310615.0 5205055.5, 1301479.5 5214191.0, 1292344.0 5205055.5))"
    }
)
```

As you can see geometry has to be specified as a string in wkt format (the same way as when calling Sentinel Hub services directly). Also coordinates have to be in the same CRS as the `BBox`.


#### Improvement for the future:
We could add wrapper classes for geometrical elements (e.g. `class Polygon`). Then geometry could be specified like this:

`CustomUrlParam.GEOMETRY: Polygon(...)` 

where `Polygon` would handle geometry parsing and CRS transformations.

That would take more time to implement therefore I am closing this issue for now.

**Comment by saanasum** [comment]
Thank you!
